7_react/react1/week2/my-app/src/TodoItem/TodoItem.js

```diff
@@ -0,0 +1,18 @@
+import "./TodoItem.css"
+
+export default function TodoItem(props) {
+    const decorateDone = props.done ? 
+        <span><s>{props.description}</s></span> 
+        :<span>{props.description}</span>
+
+    return (
+        <li className="todoItem">
+            {decorateDone}
+            <input
```

Comment:
For screen readers (people with limited vision use these to "see" websites), we should always have a `label` or an `aria-label` on input fields. 
More info on labels: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label